dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server

```diff
@@ -1171,12 +1177,9 @@ restoreWorkarea()
   if [ ! -f "${SERVER_OUTPUT_DIR}/workarea/checkpoint/.restoreMarker" ]; then
     return 0
   fi
-  if [ ! -d "${SERVER_OUTPUT_DIR}/workarea-checkpoint" ]; then
+  if [ ! -d "${SERVER_OUTPUT_DIR}/workarea/checkpoint/workarea-checkpoint" ]; then
     return 1
   fi
-  if [ ! -d "${SERVER_OUTPUT_DIR}/workarea/checkpoint" ]; then
```

Comment:
Did you think this was a duplicate check from above?  Or are you counting on the mv of checkpoint to fail below?  I guess having the mv fail will work also.

Reply:
That check is now redundant. If the check in the **NEW code** immediately above passes:
```
  if [ ! -d "${SERVER_OUTPUT_DIR}/workarea/checkpoint/workarea-checkpoint" ]; then
    return 1
  fi
```
then `-d "${SERVER_OUTPUT_DIR}/workarea/checkpoint"` is true.